### ISSUE TITLE
Misleading error message about the supported Rocket.Chat version

### ISSUE THREAD

**Issue by Gummikavalier** [RocketChat/Rocket.Chat.Electron#2853]
### Search before asking

- [X] I had searched in the [issues](https://github.com/RocketChat/Rocket.Chat.Electron/issues?q=is%3Aissue) and found no similar issues.

### Operating System

- [X] macOS
- [X] Windows
- [ ] Linux

### Operating System Version

Windows 11, MacOS

### It happens on the web browser too?

No, it just happens on the Desktop app

### Rocket.Chat Desktop App Version

3.9.11

### Rocket.Chat Server Version

6.6.1

### Describe the bug

After the RC server version update the users of older RC desktop client users get an error message that indicates that the server needs to be updated, instead of a message that the client needs to be updated.

### How to Reproduce

1. Install client version 3.9.11
2. Connect to RC 6.5.x
3. Update RC on the server to 6.6.1

### Describe your Expected behavior

Users should get a message that informs them about the need to update their clients.

Instead of correct measure to fix their issue, they get a message that indicates completely opposite:
![screenshot-21022024-0913](https://github.com/RocketChat/Rocket.Chat.Electron/assets/6958942/2ce72ba6-53b7-4e44-9358-63d51b2b4aaa)

### Are you willing to submit a code contribution?

- [ ] Yes, I am willing to submit a Pull Request!

**Comment by Gummikavalier** [comment]
Important note that may affect his issue is that RC 6.6.1 are shown as unsupported at the server end too:
![Screenshot from 2024-02-21 10-39-30](https://github.com/RocketChat/Rocket.Chat.Electron/assets/6958942/b2913ba7-5576-40a7-b154-fd9a8fa67e48)

Related issue in the RC server side:
https://github.com/RocketChat/Rocket.Chat/issues/31706

**Comment by Gummikavalier** [comment]
It is likely that this issue is caused by the faulty server end version status reporting particularly in loosely network limited systems that use offline license. Also community editions are affected in the same conditions.

**Comment by florisvangeel** [comment]
This is actually not a bug but a (marketing) feature, we see it defined [here](https://github.com/RocketChat/Rocket.Chat.Electron/blob/b4aeef45bdfb7b0b74375697543cc6c9c83ad866/src/urls.ts#L18) the client make a query to this json endpoint and starts nagging in toasters, then even disables access to the `unsupported` server.

This has nothing to do with legacy or CVE bugs and is deliberately put in the code.

**Comment by Gummikavalier** [comment]
It has been the norm that RC developers do not much shout about of their existing security vulnerabilities and keeping up to date is pretty much mandatory if one cares at all about their security.

I agree that there has been a clear marketing edge. But probably also a development roadmap changes causing this particular issue. (Getting rid of old code and methods etc.)

I also agree that enforcing people to move to versions that limit existing options and causes extra work and problems for everybody, including paying customers, is bad custom.

But since the original issue I reported here has been fixed on both server and client sides, I close this issue now. (I had forgot to do so.)
